packages/dx-grid-core/src/plugins/_blueprint/computeds.ts

```diff
@@ -0,0 +1,9 @@
+// tslint:disable-next-line: space-in-parens
+export const pureComputed = (arg: any/* param1, param2 */) => {
```

Comment:
I this function need?

Reply:
It's a 'best practice' of writing such functions, so I left it as is

Reply:
Maybe we should not change these files? Keep `.js` and our lint will skip these files. Add comments `disable ts lint` will not need.